src/gentropy/datasource/gwas_catalog/associations.py

```diff
@@ -988,11 +919,177 @@ def _qc_palindromic_alleles(
             StudyLocusQualityCheck.PALINDROMIC_ALLELE_FLAG,
         )
 
+    @staticmethod
+    def _get_effect_type(ci_text: Column) -> Column:
```

Comment:
Do you know more or less how many nulls we have here? Just to understand if the approach of using increase/decrease to distinguish works

Reply:
That's what i wanted to know, thanks!